samples/kspsample/src/commonTest/kotlin/com/careem/mockingbird/kspsample/KspSample2Test.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Careem, an Uber Technologies Inc. company
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.careem.mockingbird.kspsample
+
+import com.careem.mockingbird.test.annotations.Mock
+import kotlin.test.Test
+import kotlin.test.assertNotNull
+
+class KspSample2Test {
```

Comment:
IMHO you could remove UsedOnce mock and just write description in the test to explain why we have this test

Reply:
addressed